specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/postgresql.json

```diff
@@ -1492,6 +1490,13 @@
           "x-ms-client-flatten": true,
           "description": "Properties of the server."
         },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        },
```

Comment:
`tags` has been defined in common-types/resource-management/v1/types.json#/definitions/TrackedResource, so no need to define it in your model, which causes autorest failure.

Reply:
Thank you so much for pointing it out! I removed it, and updated PR description. Please take a look.